### DISCUSSION TITLE
[bug吗？] /get_guild_members无法使用

### DISCUSSION THREAD

**Discussion by GLGDLY** [Mrs4s/go-cqhttp#1234] [Q&A] [answered]
/get_guild_members 会回报API不存在。我在尝试以链接
http://127.0.0.1:5700/get_guild_members?guild_id=8xxxxxxxxxxxxxxx7
获取频道成员消息时出现错误

- 频道id存在且可以发送频道消息——当相同方法发送链接
http://127.0.0.1:5700/send_guild_channel_msg?guild_id=8xxxxxxxxxxxxxxx7&channel_id=1xxxxx0&message=xxx
 时可以反馈{"data":{"message_id":"1xxxxx-xxxxxxx1"},"retcode":0,"status":"ok"}

版本beta-8 fix2，环境1Android，无相应日志产生

**Comment by Ink-33** [comment, marked as answer]
这个api暂时有点问题，移除一段时间，过段时间会加上

**Reply by GLGDLY** [reply]
ok明白谢谢

**Comment by Mrs4s** [comment]
下个版本会加回来, 之前有问题
